### PR TITLE
MAINT: Dictionary litteral

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: numpy/core/tests/test_item_selection.py [hunk 1]
```diff
@@ -18,10 +18,9 @@ def test_simple(self):
         index_arrays = [np.empty(0, dtype=np.intp),
                         np.empty(tuple(), dtype=np.intp),
                         np.empty((1, 1), dtype=np.intp)]
-        real_indices = {}
-        real_indices['raise'] = {-1:1, 4:IndexError}
-        real_indices['wrap'] = {-1:1, 4:0}
-        real_indices['clip'] = {-1:0, 4:1}
+        real_indices = {'raise': {-1: 1, 4: IndexError},
```
Comment (jaimefrio) [inline numpy/core/tests/test_item_selection.py]: I think the convention is no spaces in slices, I.e. after the colon.

Comment (njsmith) [inline numpy/core/tests/test_item_selection.py]: Oh wow, that is confusing. I had to read like 5 times to see it, but
they're not slices, they're dict key/values.

Comment (sieben) [inline numpy/core/tests/test_item_selection.py]: Is it fine then? Should it be merged?

Comment (jaimefrio) [inline numpy/core/tests/test_item_selection.py]: Indeed they are, definitely no change needed...

**Comment by jaimefrio** [comment]
Thanks, and sorry for the confusion.

**Pull request merged**